deno.json

```diff
@@ -1,29 +1,27 @@
 {
   "tasks": {
-    "chel": "deno task build && deno run --import-map=./vendor/import_map.json --allow-net --allow-read=. --allow-write=. --no-remote build/main.js",
+    "chel": "deno task build && deno run --allow-net --allow-read=. --allow-write=. build/main.js",
     "lint": "standard ./src",
-    "vendor": "deno vendor -f src/main.ts scripts/compile.ts",
-    "compile": "deno run --import-map=./vendor/import_map.json --no-remote --allow-run --allow-read=. --allow-write=./dist scripts/compile.ts",
-    "build": "deno run --import-map=./vendor/import_map.json --no-remote --allow-run --allow-read --allow-env --allow-write=./build --allow-net scripts/build.ts",
-    "dist": "deno lint && deno task build && deno task compile",
-    "test": "deno test --import-map=./vendor/import_map.json --allow-read=. --allow-write=test --no-remote"
+    "compile": "deno run --allow-run --allow-read=. --allow-write=./dist scripts/compile.ts",
+    "build": "deno run --node-modules-dir --allow-run --allow-read --allow-env --allow-write=./build --allow-net scripts/build.ts",
+    "dist": "deno lint . && deno task build && deno task compile",
+    "test": "deno test --allow-read=. --allow-write=test"
   },
   "imports": {
-    "https://deno.land/std/io/util.ts": "https://deno.land/std@0.156.0/io/util.ts"
+    "~/": "./src/",
+    "jsr:@std/": "https://jsr.io/@std/",
+    "npm:@types/node": "npm:@types/node@latest"
   },
   "compilerOptions": {
-    "allowJs": true,
-    "strict": true
   },
+  "nodeModulesDir": "auto",
   "fmt": {
     "lineWidth": 100,
     "semiColons": false,
     "singleQuote": true
   },
   "lint": {
-    "files": {
-      "exclude": ["node_modules/", "test/", "vendor/", "dist/", "build/"]
-    }
+    "exclude": ["node_modules/", "vendor/", "dist/", "build/", "test/assets/**/*.js"]
```

Comment:
I don't see a need for excluding `js` files in particular. `test/assets/` should be good enough, since no files whatsoever under this directory should be linted.

Reply:
https://github.com/okTurtles/chel/pull/46/commits/54a9fa20b38171178935f7d056540a002ac32a22